report/tsconfig.json

```diff
@@ -0,0 +1,39 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "../.tmp/tsbuildinfo/report",
+    "emitDeclarationOnly": true,
+    "declarationMap": true,
+
+    // Limit to base JS and DOM defs.
+    "lib": ["es2020", "dom", "dom.iterable"],
+    // Include `@types/node` for use of `Buffer` in text-encoding.js.
+    "types": ["node"],
+    "target": "es2020",
+    "module": "es2020",
+    "moduleResolution": "node",
+
+    "allowJs": true,
+    "checkJs": true,
+    "strict": true,
+    // TODO: remove the next line to be fully `strict`.
+    "useUnknownInCatchVariables": false,
+
+    // "listFiles": true,
+    // "noErrorTruncation": true,
+    "extendedDiagnostics": true,
+  },
+  "include": [
+    "**/*.js",
+    "types/**/*.d.ts",
+  ],
+  "references": [
+    {"path": "../types/lhr/"},
+    {"path": "./generator/"}
+  ],
+  "exclude": [
+    "generator/**/*.js",
```

Comment:
I didn't realize `generator/*` files would be included even though `generator/` is in `references` and has its own `tsconfig`. Having to explicitly exclude these is one more argument not to nest `generator` under `report`, I think (see other options in https://github.com/GoogleChrome/lighthouse/pull/12940#issue-716094883).